nservicebus/messaging/send-a-message.md

```diff
@@ -1,43 +1,54 @@
 ---
-title: Sending a Message
-summary: Describes how to send a message
+title: Sending messages
+summary: Describes how to send messages
 tags: []
 redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-Message sending involves using Send operation that takes an argument of a message instance to be delivered. The details differ slightly between version of NServiceBus, as shown in following snippets.
+NServiceBus supports sending different types of messages (see [Messages, Events and Commands](messages-events-commands.md)) to any endpoint. Messages can be sent from outside or inside the message processing pipeline:
```

Comment:
Can we provide some more context about the pipeline here? For someone that doesn't know what it is we tell them it's important without really giving any context. Something like this:

NServiceBus supports sending different types of messages (see [Messages, Events and Commands](messages-events-commands.md)) to any endpoint. _When a message arrives at an endpoint it goes through a pipeline of processing steps_. Messages can be sent from outside or inside the message processing pipeline: